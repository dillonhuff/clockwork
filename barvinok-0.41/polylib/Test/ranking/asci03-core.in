4 4
 1 -1 0 5
 1 0 1 -1
 1 0 -1 4
 1 1 0 -1

4 4
 1 -1 0 6
 1 0 1 -1
 1 0 -1 4
 1 1 0 -2

0 2
