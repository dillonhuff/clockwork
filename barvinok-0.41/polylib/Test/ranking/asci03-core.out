============ Ranking function ============
         Q  -1 = 0
         P  -2 >= 0
         - P + 6 >= 0

UNION
         P  -6 = 0
         Q  -1 >= 0
         - Q + 4 >= 0

( 4 * P + -4 )

-----------------------------------
         P  -2 >= 0
         - P + 5 >= 0
         Q  -2 >= 0
         - Q + 4 >= 0

( 1 * Q + ( 4 * P + -5 )
 )

-----------------------------------
